Retrie/Options.hs

```diff
@@ -213,8 +222,20 @@ buildParser dOpts = do
   rewrites <- parseRewriteSpecOptions
   elaborations <- parseElaborations
   roundtrips <- parseRoundtrips
+  defaultExtensions <- parseExtensions
   return Options{ fixityEnv = fixityEnv dOpts, ..}
 
+-- ugh
+deriving stock instance Read Extension
```

Comment:
GHC PR: https://gitlab.haskell.org/ghc/ghc/-/merge_requests/7768

I imagine they'll either OK it (and we can make this conditional at some point) or point me in some other direction, which works alright for me.